shell/platform/darwin/ios/ios_context_gl.mm

```diff
@@ -43,6 +43,16 @@
       GrBackend::kOpenGL_GrBackend, GPUSurfaceGLDelegate::GetDefaultPlatformGLInterface());
 }
 
+// |IOSContext|
+sk_sp<GrDirectContext> IOSContextGL::GetMainContext() const {
+  /// TODO(b/tbd): Currently the GPUSurfaceGL creates the main context for
```

Comment:
I'll write up the issue once this PR has been approved.  This means the savings won't happen on OpenGL until that PR is resolved.  It was risky enough and lower priority since less people are on OpenGL so I punted for now.